arm-customer-insights/2016-01-01/swagger/customer-insights.json

```diff
@@ -0,0 +1,4735 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "AciManagementClient",
+    "description": "The Azure Customer Insights management API provides a RESTful set of web services that interact with Azure Customer Insights service to manage your resources. The API has entities that capture the relationship between an end user and the Azure Customer Insights service.",
+    "version": "2016-01-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}": {
+      "put": {
+        "tags": [
+          "Hubs"
+        ],
+        "operationId": "Hubs_CreateOrUpdate",
+        "x-ms-examples": { 
+          "Hubs_CreateOrUpdate": { "$ref": "../examples/HubsCreateOrUpdate.json" } 
+        },        
+        "description": "Creates a hub, or updates an existing hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Hub."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Hub"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate Hub operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
The RP does return 409 for this case, but do we need to list all the non success HTTP status code here? Wouldn't them be handled in a default way?
Or we just need 409 here particularly?